lib/zstd.h

```diff
@@ -533,452 +465,501 @@ ZSTDLIB_API size_t ZSTD_frameHeaderSize(const void* src, size_t srcSize);
+                              * while compression work is performed in parallel, within worker threads.
+                              * (note : a strong exception to this rule is when first invocation sets ZSTD_e_end : it becomes a blocking call).
+                              * More workers improve speed, but also increase memory usage.
+                              * Default value is `0`, aka "single-threaded mode" : no worker is spawned, compression is performed inside Caller's thread, all invocations are blocking */
+    ZSTD_p_jobSize=401,      /* Size of a compression job. This value is enforced only when nbWorkers >= 1.
+                              * Each compression job is completed in parallel, so this value can indirectly impact the nb of active threads.
+                              * 0 means default, which is dynamically determined based on compression parameters.
+                              * Job size must be a minimum of overlapSize, or 1 MB, whichever is largest.
+                              * The minimum size is automatically and transparently enforced */
+    ZSTD_p_overlapSizeLog=402, /* Size of previous input reloaded at the beginning of each job, as a fraction of window size.
+                              * 0 : no overlap;  6(default) : use 1/8th of windowSize;  >=9 : use full windowSize */
 
+    /* =================================================================== */
+    /* experimental parameters - no stability guaranteed                   */
+    /* => note : should this part be exported in a different section of zstd.h ? */
+    /* =================================================================== */
 
+    /* compression format */
+    ZSTD_p_format = 10,      /* See ZSTD_format_e enum definition.
+                              * Cast selected format as unsigned for ZSTD_CCtx_setParameter() compatibility. */
 
-/***************************************
-*  Advanced compression functions
-***************************************/
+    ZSTD_p_forceMaxWindow=1100, /* Force back-reference distances to remain < windowSize,
+                              * even when referencing into Dictionary content (default:0) */
+    ZSTD_p_forceAttachDict,  /* Controls whether the contents of a CDict are
+                              * used in place, or whether they are copied into
+                              * the working context.
+                              *
+                              * Accepts values from the ZSTD_dictAttachPref_e
+                              * enum. See the comments on that enum for an
+                              * explanation of the feature.
+                              */
+    /* Question : should rsyncable be categorized as experimental, or be candidate for stable ? */
+    ZSTD_p_rsyncable,        /* Enables rsyncable mode, which makes compressed
```

Comment:
I think this could be a candidate for stable, but its also fine to leave it in experimental for a release or two.

Reply:
Associated question :
does moving an `enum` from `experimental` to `stable` increases the minor release number (`v1.4.0` -> `v1.5.0`) everything else being equal ?